wordle-web/components/ConfirmDeleteWord.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <v-dialog v-model="modelValue" max-width="500" persistent>
```

Comment:
This looks great! If I wasn't super crunched on time I would try to see what it looks like, but I like the idea :)